src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -336,17 +382,19 @@ class ArnoldRendererBase : public IECoreScenePreview::Renderer
 
 		ObjectInterfacePtr camera( const std::string &name, const IECoreScene::Camera *camera, const AttributesInterface *attributes ) override;
 		ObjectInterfacePtr light( const std::string &name, const IECore::Object *object, const AttributesInterface *attributes ) override;
+		ObjectInterfacePtr lightBlocker( const std::string &name, const IECore::Object *object, const AttributesInterface *attributes ) override;
 		Renderer::ObjectInterfacePtr object( const std::string &name, const IECore::Object *object, const AttributesInterface *attributes ) override;
 		ObjectInterfacePtr object( const std::string &name, const std::vector<const IECore::Object *> &samples, const std::vector<float> &times, const AttributesInterface *attributes ) override;
 
 	protected :
 
-		ArnoldRendererBase( NodeDeleter nodeDeleter, AtNode *parentNode = nullptr );
+		ArnoldRendererBase( NodeDeleter nodeDeleter, BlockerConnections *blockerConnections, AtNode *parentNode = nullptr );
```

Comment:
60efade130e878cec576ef060570928cf96e01c2

Maybe we can talk about this when we skype the next time because I wasn't quite sure about the reference part of it. I've made it a regular smart pointer for now.